### ISSUE TITLE
Relationship fields are missing column view and are displayed as invalid field type column

### ISSUE THREAD

**Issue by tomasztunik** [keystonejs/keystone#1617]
In 0.3.x relationship fields were valid for displaying in column view and would link to the related models by displaying name or id if name or name alias was not specified.

Currently only the first field links to the document which is listed and there are no linkTo generated for other fields to be used. I've already implemented most of it but put it on hold for now. Wanted to ask first about what would be the best way to populate relationship type fields on the document so that we could use name field/alias or Id if no name is set as a label for relationship link. Currently all the data to generate link is available but the we have only Id of the related field. Or maybe i'm missing something and it's already possible to retrieve this data if so I'd appreciate for a tip on how to get it :)

We also need to remember about many: true relationships and generate proper list of links for all the related models.

if I knew how to get to populate the models in the cleanest way I'm happy to work on it if anybody haven't started working on it yet.

**Comment by JedWatson** [comment]
@tomasztunik sorry for the confusion around this, was on my todo list still. the functionality was waiting on #1544 to land, which gave us the data we need to properly render the relationships in the list.

I've just committed an implementation for this, so it should work as before now - let me know if you can see any other problems with it.
